st2actions/st2actions/query/mistral/v2.py

```diff
@@ -42,13 +42,25 @@ def query(self, execution_id, query_context):
             status, output = self._get_workflow_result(exec_id)
             if output and 'tasks' in output:
                 LOG.warn('Key conflict with tasks in the workflow output.')
+        except requests.exceptions.ConnectionError:
+            msg = 'Unable to connect to mistral.'
```

Comment:
I can add the traceback but it is pretty useless since we anyways know it's a connection error.